### ISSUE TITLE
Fucked up TFA/Weapon shit. In Urgent need of help.

### ISSUE THREAD

**Issue by Payday254** [Zet0rz/nzombies#654]
Video of my issue.
This happens everytime I load.
This is some shortened shit outta my console.

: revive_system/sh_playerdeath.lua
Including: revive_system/sh_revival.lua
Including: revive_system/sh_sync.lua
Constructing: round
Including: round/sh_constructor.lua
Including: round/cl_fog.lua
Including: round/cl_round.lua
Including: round/cl_round_meta.lua
Including: round/sh_boss_round.lua
Including: round/sh_player.lua
Including: round/sh_special_round.lua
Including: round/sh_sync.lua
Including: scoring/sh_player.lua
Constructing: special_weapons
Including: special_weapons/sh_constructor.lua
Including: special_weapons/sh_functions.lua
Including: special_weapons/sh_specialweapons.lua
Including: special_weapons/sh_sync.lua
Constructing: tools
Including: tools/sh_constructor.lua
Including: tools/sh_meta.lua
Including: tools/sh_sync.lua
Including: tools/sh_tools.lua
Including: tools/sh_tools_barricade.lua
Including: tools/sh_tools_block.lua

Including: tools/sh_tools_zspecialspawn.lua
Including: tools/sh_traps_logic.lua
Constructing: traps_logic
Including: traps_logic/sh_constructor.lua
Constructing: weapons
Including: weapons/sh_constructor.lua

[Black Ops II Weapons Pack: Remastered] lua/weapons/bo2r_870mcs/sh_sounds.lua:1: attempt to index global 'CustomizableWeaponry' (a nil value)
1. unknown - lua/weapons/bo2r_870mcs/sh_sounds.lua:1
2. include - [C]:-1
3. unknown - lua/weapons/bo2r_870mcs/shared.lua:3

SWEP (khr_t5000) is derived from non existant SWEP (cw_base) - Expect errors!

[[CW 2.0] PPSh-41] lua/entities/cw_ammo_762x25/shared.lua:1: attempt to index global 'CustomizableWeaponry' (a nil value)
1. unknown - lua/entities/cw_ammo_762x25/shared.lua:1
2. include - [C]:-1
3. unknown - lua/entities/cw_ammo_762x25/cl_init.lua:1

[[CW 2.0] Mosin Nagant] lua/entities/cw_ammo_762x54mmr/shared.lua:1: attempt to index global 'CustomizableWeaponry' (a nil value)
1. unknown - lua/entities/cw_ammo_762x54mmr/shared.lua:1
2. include - [C]:-1
3. unknown - lua/entities/cw_ammo_762x54mmr/cl_init.lua:1

[[CW 2.0] Sako TRG-42] lua/entities/cw_ammo_lupa/shared.lua:1: attempt to index global 'CustomizableWeaponry' (a nil value)
1. unknown - lua/entities/cw_ammo_lupa/shared.lua:1
2. include - [C]:-1
3. unknown - lua/entities/cw_ammo_lupa/cl_init.lua:1

ERROR: Trying to derive entity cw_rinic_ext_mag from non existant entity cw_attpack_base!
ERROR: Trying to derive entity khr_attpack_newslugrounds from non existant entity cw_attpack_base!
ERROR: Trying to derive entity cw_ammo_hk4.6x30mm from non existant entity cw_ammo_ent_base!
ERROR: Trying to derive entity khr_attpack_mosinstyles from non existant entity cw_attpack_base!
ERROR: Trying to derive entity m240_bipod from non existant entity cw_attpack_base!
ERROR: Trying to derive entity cw_ammo_30winchester from non existant entity cw_ammo_ent_base!
ERROR: Trying to derive entity cw_aug_stuff from non existant entity cw_attpack_base!
ERROR: Trying to derive entity khr_attpack_sw642barrels from non existant entity cw_attpack_base!
ERROR: Trying to derive entity cw_ammo_khr_3006 from non existant entity cw_ammo_ent_base!
ERROR: Trying to derive entity m16_stuff from non existant entity cw_attpack_base!
ERROR: Trying to derive entity khr_attpack_sw29barrels from non existant entity cw_attpack_base!
ERROR: Trying to derive entity cw_ammo_300aacblackout from non existant entity cw_ammo_ent_base!
ERROR: Trying to derive entity khr_attpack_judgebarrels from non existant entity cw_attpack_base!
ERROR: Trying to derive entity m98b_stuff from non existant entity cw_attpack_base!
ERROR: Trying to derive entity cw_ammo_357magnum from non existant entity cw_ammo_ent_base!
ERROR: Trying to derive entity khr_attpack_sksbumpfire from non existant entity cw_attpa

This map has been SPOOKIFIED.

xture" which is not a texture value (material: skybox/paintedrt)

1	=	59
2	=	60
3	=	61
grenade	=	57
specialgrenade	=	58
Received Full Perks Sync
Video of my issues.
https://www.youtube.com/watch?v=itTlzOLodFo

**Comment by Payday254** [comment]
And it also had some "ammo1" issue.

**Comment by DjNathann** [comment]
Try using TFA Base instead CW 2.0

**Comment by Payday254** [comment]
What if I have both?
@DjNathann

**Comment by Payday254** [comment]
@Zet0rz Is this a known issue?

**Comment by minin43** [comment]
There is no conflict with the CW2.0 weapon base and the gamemode. Any problems you are having are because you have done something incorrectly.

From the looks of the errors you're printing, *you don't have the CW2.0 weapon base installed on your client/server.*

Here's the link to it: https://steamcommunity.com/sharedfiles/filedetails/?id=349050451

**Comment by Payday254** [comment]
Nope. I have it installed.
minin this has been going for days, I've been checking and there's "no" addon i'm missing as far as I know

**Comment by minin43** [comment]
No. I'm telling you, for some reason or another, you don't have the addon running in your game when you play. Subscribe to the weapon base.

I'm a developer. I'm looking at the errors as I type this. That is 100% your problem.

Additionally, there is no conflict with the CW2.0 weapon base and this gamemode. I have used it for *literally* years and never had a problem.

**Comment by Payday254** [comment]
Nope. I have it subscribed. I dont know what the issue is.

**Comment by Payday254** [comment]
@minin43 nope. nothing. Ive tried new maps and everything. additionally when I load a config it says theres a bunch of shit not"configured"

**Comment by minin43** [comment]
@Payday254 Then you're out of luck. Reinstall garry's mod if you have to.

**Comment by JustAnotherRookie** [comment]
Try reinstalling both TFA and nZombies, along with all of your TFA addons, I had to do this when a similar issue prevented any of my weapons from loading.
